tests/expected/HumanR2022a.proto

```diff
@@ -0,0 +1,939 @@
+#VRML_SIM R2022a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# This is a proto file for Webots for the HumanR2022a
+# Extracted from: /home/benja/urdf2webots/tests/sources/gait2392_simbody/urdf/human.urdf
```

Comment:
```suggestion
# Extracted from: /home/benja/urdf2webots/tests/sources/gait2392_simbody/urdf/human.urdf
```
Why it contains a path on my computer ? I do not remember even pushing stuff on this repo other than version number update.